torchmetrics/functional/text/bert.py

```diff
@@ -450,8 +451,10 @@ def _rescale_metrics_with_baseline(
 
 
 def bert_score(
-    predictions: Union[List[str], Dict[str, Tensor]],
-    references: Union[List[str], Dict[str, Tensor]],
+    preds: Union[List[str], Dict[str, Tensor]],
+    target: Union[List[str], Dict[str, Tensor]],
+    predictions: Union[None, List[str], Dict[str, Tensor]] = None,
+    references: Union[None, List[str], Dict[str, Tensor]] = None,
```

Comment:
the old one shall be at the very end, imagine that you have a user who uses only positional arguments, this will break for him... :rabbit:

Reply:
Ohh, that's true. I'm gonna fix that :]